desktop/src/main/java/bisq/desktop/primary/main/content/social/chat/ChatController.java

```diff
@@ -273,10 +273,19 @@ public void onReply(ChatMessage chatMessage) {
         }
     }
 
-    public void onSendPrivateMessage(ChatMessage chatMessage) {
-        if (!model.isMyMessage(chatMessage)) {
-            // todo
+    /**
+     * open a private channel to specified user. Automatically select it so user is ready to type the message to send.
+     * @param chatMessage
+     */
+    public void addPrivateChannel(ChatMessage chatMessage) {
```

Comment:
I use the on* convention for UI handlers (e.g. onAddPrivateChannel). Makes it more clear in which context the method is used.